pinecone/index_connection_test.go

```diff
@@ -373,6 +359,82 @@ func (ts *IndexConnectionTestsIntegration) TestMetadataAppliedToRequests() {
 	require.NotNil(ts.T(), stats)
 }
 
+func (ts *IndexConnectionTestsIntegration) TestUpdateVectorValues() {
+	ctx := context.Background()
+
+	expectedVals := []float32{7.2, 7.2, 7.2, 7.2, 7.2}
+	err := ts.idxConn.UpdateVector(ctx, &UpdateVectorRequest{
+		Id:     ts.vectorIds[0],
+		Values: expectedVals,
+	})
+	assert.NoError(ts.T(), err)
+
+	time.Sleep(5 * time.Second)
```

Comment:
Has 5 seconds worked well for this so far? We had use retries and longer wait windows in some other tests to handle upsert and update. Just curious how it's performed for you so far.

Reply:
Yeah I tried longer and shorter, but 5 was the shortest I found where it still passed